templates/patient/insurance/insurance_edit.html.twig

```diff
@@ -0,0 +1,425 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+    {% block scripts %}
+    {% endblock %}
+
+    <style>
+        div.insuranceEditContainer div.label_custom span {
+            font-size: 0.8rem;
+            display: inline-flex;
+            height: 100%;
+            align-items: center;
+            line-height: 1.2;
+        }
+
+        {% if rightJustifyLabels and languageDirection == 'ltr' %}
+            div.label_custom {
+              text-align: right !important;
+            }
+
+            div.tab td.data, div.data {
+                padding-left: 0.5em;
+                padding-right: 2em;
+            }
+        {% endif %}
+    </style>
+</head>
+<body class="body_top">
+{% block preContent %}{% endblock %}
+{% block content %}
+
+{% set dataTypeSelectList = 1 %}
+{% set dataTypeText = 2 %}
+{% set dataTypeDate = 4 %}
+
+{# Note that date fields are handled in javascript because we are building a template instead of handling it all via php  #}
+{# Note that empty_title: 'SKIP' is a hack in the function to skip a title... terrible but is the existing code unfortunately in generate_form_field. #}
+{% set fields = {
+    left: [{
+        name: "Plan Name"|xl
+        ,required: false
+        ,data_type: dataTypeText
+        ,field_id: "plan_name"
+        ,fld_length: 20
+        ,max_length: 63
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1 js-swap-insurance"
+    }
+    ,{
+        name: "Effective Date"|xl
+        ,required: true
+        ,data_type: dataTypeDate
+        ,field_id: "date"
+        ,fld_length: 16
+        ,max_length: 60
+        ,smallform: 'true'
+        ,classNames: "datepicker mb-1"
+        ,autocomplete: 'off'
+    },{
+        name: "Effective Date End"|xl
+        ,required: false
+        ,data_type: dataTypeDate
+        ,field_id: "date_end"
+        ,fld_length: 16
+        ,max_length: 60
+        ,smallform: 'true'
+        ,classNames: "datepicker mb-1"
+        ,autocomplete: 'off'
+    }, {
+        name: "Policy Number"|xl
+        ,required: true
+        ,data_type: dataTypeText
+        ,field_id: "policy_number"
+        ,fld_length: 16
+        ,max_length: 60
+        ,smallform: 'true'
+        ,classNames: "js-policykeyup mb-1"
+    }, {
+        name: "Group Number"|xl
+        ,required: false
+        ,data_type: dataTypeText
+        ,field_id: "group_number"
+        ,fld_length: 16
+        ,max_length: 60
+        ,smallform: 'true'
+        ,classNames: "js-policykeyup mb-1"
+    },{
+        name: "Subscriber Employer (SE)"|xl
+        ,required: false
+        ,data_type: dataTypeText
+        ,field_id: "subscriber_employer"
+        ,fld_length: 25
+        ,max_length: 60
+        ,smallform: 'true'
+        ,classNames: "js-capitalize-me mb-1"
+        ,skip: include_employers
+        ,description: 'if unemployed enter Student'|xl ~','~'PT Student, or leave blank'|xl
```

Comment:
note sure what the `~` does here. but in theory should be using xlj here instead of xl

Reply:
Its a twig concatenate operation and this is the description field in generate_form_field so I just translate it instead of escaping it.